lsp-mode.el

```diff
@@ -6260,15 +6260,15 @@ A reference is highlighted only if it is visible in a window."
        (-map
         (-lambda ((start-window . end-window))
           ;; Make the overlay only if the reference is visible
-          (let ((start-point (lsp--position-to-point start))
```

Comment:
I think `(lsp--position-to-point start)` and `(lsp--position-to-point end)` is a heavy job.